src/Hl7.Fhir.ElementModel/MaskingNode.cs

```diff
@@ -152,6 +154,6 @@ public IEnumerable<object> Annotations(Type type)
         }
 
         public IEnumerable<ITypedElement> Children(string name = null) =>
-            Source.Children(name).Where(c => included(c)).Select(c => new MaskingNode(this, c));
+            Source.Children(name).Where(c => included(c)).Select(c => new MaskingNode(this, c, _settings.IncludeMandatory && !Source.Location.StartsWith("Bundle") ? (bool?)true : (bool?)null));
```

Comment:
This seems to suggest ANY (nested) child of Bundle will be exempt, is that correct?  Or did you mean the direct children of Bundle (=1 layer deep)?  The StartsWith() should probably include "Bundle.", to make sure you don't match "BundledMedication" (or such new resource names).